src/BlazorUI/Bit.BlazorUI.Tests/Buttons/BitIconButtonTests.cs

```diff
@@ -5,7 +5,7 @@
 namespace Bit.BlazorUI.Tests.Buttons;
 
 [TestClass]
-public class BitIconButtonTests : BunitTestContext
+public class BitIconButtons : BunitTestContext
```

Comment:
This is the same as before, you should put the "Tests" at the end of the class name.

Reply:
done